config.go

```diff
@@ -1,267 +1,268 @@
 // * a pointer to the function that manages the switch
 // * the configuration that gets updated
-func push_cmd_line_item(Switch string, Help_str string, Has_arg bool, Needed bool, Func Arg_func, cmd_line *[]cmd_line_items){
-	*cmd_line = append(*cmd_line, cmd_line_items{id: len(*cmd_line)+1, Switch: Switch, Help_srt: Help_str, Has_arg: Has_arg, Needed: Needed, Func: Func})
+func pushCmdLineItem(switchStr string, helpStr string, hasArg bool, needed bool, function argFunc, cmdLine *[]cmdLineItems) {
+	*cmdLine = append(*cmdLine, cmdLineItems{id: len(*cmdLine) + 1, switchStr: switchStr, helpStr: helpStr, hasArg: hasArg, needed: needed, function: function})
 }
 
 // This function initializes configuration parser subsystem
-// Inserts all the commandline switches suppported by the application
-func cmd_line_item_init() ([]cmd_line_items){
-	var res	[]cmd_line_items
+// Inserts all the commandline switches supported by the application
+func cmdLineItemInit() []cmdLineItems {
+	var res []cmdLineItems
 
-	push_cmd_line_item("-j", "Force Json output with subsystems data",	true,  false,	func_outtype,	&res)
-	push_cmd_line_item("-s", "Specifies symbol",				true,  true,	func_symbol,	&res)
-	push_cmd_line_item("-i", "Specifies instance",				true,  true,	func_instance,	&res)
-	push_cmd_line_item("-f", "Specifies config file",			true,  false,	func_jconf,	&res)
-	push_cmd_line_item("-u", "Forces use specified database userid",	true,  false,	func_DBUser,	&res)
-	push_cmd_line_item("-p", "Forecs use specified password",		true,  false,	func_DBPass,	&res)
-	push_cmd_line_item("-d", "Forecs use specified DBhost",			true,  false,	func_DBHost,	&res)
-	push_cmd_line_item("-p", "Forecs use specified DBPort",			true,  false,	func_DBPort,	&res)
-	push_cmd_line_item("-m", "Sets display mode 2=subsystems,1=all",	true,  false,	func_Mode,	&res)
-	push_cmd_line_item("-h", "This Help",					false, false,	func_help,	&res)
+	pushCmdLineItem("-j", "Force Json output with subsystems data", true, false, funcOutType, &res)
+	pushCmdLineItem("-s", "Specifies Symbol", true, true, funcSymbol, &res)
+	pushCmdLineItem("-i", "Specifies Instance", true, true, funcInstance, &res)
+	pushCmdLineItem("-f", "Specifies config file", true, false, funcJconf, &res)
+	pushCmdLineItem("-u", "Forces use specified database userid", true, false, funcDBUser, &res)
+	pushCmdLineItem("-p", "Forces use specified password", true, false, funcDBPass, &res)
+	pushCmdLineItem("-d", "Forces use specified DBHost", true, false, funcDBHost, &res)
+	pushCmdLineItem("-p", "Forces use specified DBPort", true, false, funcDBPort, &res)
+	pushCmdLineItem("-m", "Sets display Mode 2=subsystems,1=all", true, false, funcMode, &res)
+	pushCmdLineItem("-h", "This Help", false, false, funcHelp, &res)
 
 	return res
 }
 
-func func_help		(conf *configuration,fn []string)		(error){
-	return errors.New("Command Help")
+func funcHelp(conf *configuration, fn []string) error {
+	return errors.New("command Help")
```

Comment:
Is there any reason to change the string case?

Reply:
I have used the refactoring tool in my IDE for renaming, and I overlooked that it also changed the messages. So no actual reason behind it. I can change it back tomorrow

Reply:
Sorry for my previous short and not very clear message, it was typed quickly from my phone. To elaborate: 

I have changed variable names using the IDE refactoring tool and it apparently also changed the message strings in some cases, which I overlooked.

My IDE also changed the first capital letters in the error strings to lowercase to follow this rule: https://google.github.io/styleguide/go/decisions.html#error-strings

That's why there are those discrepancies - relying on automation without a keen eye  :) Thanks for catching it in your review! 

I will change the error strings so they are not so mismatched - I think having them all in lowercase is the way to go. Or do you suggest something else @alessandrocarminati ?

Reply:
@alessandrocarminati I have pushed a commit attempting to unify the strings https://github.com/alessandrocarminati/nav/pull/10/commits/690d6e6929b1408cdd23bcb11d67d403223414d7 

Feel free to offer your thoughts

Reply:
Looks good!